packages/video_player/video_player/lib/video_player.dart

```diff
@@ -528,7 +547,7 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
   /// Sets whether or not the video should loop after playing once. See also
   /// [VideoPlayerValue.isLooping].
   Future<void> setLooping(bool looping) async {
-    value = value.copyWith(isLooping: looping);
+    value = value.copyWith(isLooping: looping, isCompleted: false);
```

Comment:
Does enabling looping start playback? I would have assumed not, and if it's sitting paused and completed, enabling looping for the future doesn't seem like it should clean `isCompleted`.